woocommerce/woocommerce-gutenberg-products-block
# Categorise patterns

## What

Structure the patterns by putting them into subfolders depicting the category.

Before - full folder content doesn't even fit the screen | After - categorised patterns
-- | --
<img width="556" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/20098064/b8c64a69-0cfd-44d2-b395-eca7b878b313"> | <img width="556" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/20098064/6c8ac21f-b442-4c8e-8aaa-2b2efdabff87">


## Why

With the increasing number of patterns, I think it became harder to navigate properly in the folder structure. Putting them into subcategories allows for easier navigation and better pattern handling.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Go to Editor
2. Make sure patterns can be added and all the images are displayed correctly. Try couple of them, for example:
  - Discount Banner with Image
  - Product Details Pattern
  - 3-Column Product Row
  - Testimonials 3 Columns


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:
